zerver/openapi/zulip.yaml

```diff
@@ -10178,6 +10178,128 @@ paths:
                           }
                         description: |
                           A typical failed JSON response for when the token does not exist:
+  /users/me/android_gcm_reg_id:
+    post:
+      operationId: add-android-reg-id
+      summary: Add an Android GCM registration ID
+      tags: ["users"]
+      description: |
+        This endpoint adds an Android GCM registration ID for push notifications.
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                token:
+                  description: |
+                    The token provided by the device.
+                  type: string
+                  example: android-token
+              required:
+                - token
+      responses:
+        "200":
+          $ref: "#/components/responses/SimpleSuccess"
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/CodedError"
+                  - description: |
+                      A typical failed JSON response for when the token's length is invalid
+                      or is empty:
+                    example:
+                      {
+                        "result": "error",
+                        "msg": "Empty or invalid length token",
+                        "code": "BAD_REQUEST",
+                      }
+        "500":
+          description: Error.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/CodedError"
+                  - example:
+                      {
+                        "result": "error",
+                        "msg": "Received 500 from push notification bouncer",
+                        "code": "BAD_REQUEST",
+                      }
+                    description: |
+                      A typical failed JSON response for when 500 response is received
+                      from the push notification bouncer:
```

Comment:
Is there a reason we need to document 500/502s here? We don't for most endpoints, instead relying on the common error-handling docs.

Reply:
I have removed those error documentation.